pwndbg/heap/__init__.py

```diff
@@ -8,18 +8,8 @@
 
 heap_chain_limit = pwndbg.config.Parameter('heap-dereference-limit', 8, 'number of bins to dereference')
 
-@pwndbg.events.new_objfile
```

Comment:
I think this needs to be on new_objfile since if you do `starti` the binary will break on the first instruction of `ld.so` and `libc` will not be available / can't find symbols and the Heap stuff will not be initialized.

Reply:
But this is only an initialization of a simple struct, not related with whether the real heap is initialized:
https://github.com/pwndbg/pwndbg/blob/ae6f25a810df3b1902b1b13e6d7924d76e4de572/pwndbg/heap/ptmalloc.py#L57-L65